README.md

```diff
@@ -497,6 +497,7 @@ Table of Contents
 ## Testing
 
   * [Applitools.com](https://applitools.com/) — Smart visual validation for web, native mobile and desktop apps. Integrates with almost all automation solutions (like Selenium and Karma) and remote runners (Sauce Labs, Browser Stack). free for open source. A free tier for a single user with limited checkpoints per week.
+  * [HeadSpin.io](https://headspin.io/) — Test your mobile application or website on thousands of real device and OS. A free tier for a single user with limited checkpoints per week.
```

Comment:
I see mention of "free trial" on their site, but no pricing details or about how many per week, do you have link?